src/status_im2/contexts/chat/bottom_sheet_composer/style.cljs

```diff
@@ -0,0 +1,136 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.style
```

Comment:
The bottom sheet color is a kind of light gray when I haven't focused on the input. Do you see the same on iOS?

<img src="https://user-images.githubusercontent.com/46027/231914690-395417f2-90f8-4852-a563-d33b992855a1.png" height="500" />

Reply:
Oh I had to set the `blur-type` to be `dark` in dark-mode. Thanks for notifying me.